mne/viz/_brain/_brain.py

```diff
@@ -682,6 +682,7 @@ def setup_time_viewer(self, time_viewer=True, show_traces=True):
             with _qt_disable_paint(self.plotter):
                 with self._ensure_minimum_sizes():
                     self.show()
+            self._update()
```

Comment:
This will only happen if we call `setup_time_viewer`, which only happens if `time_viewer=True` (effectively) is used, right? This suggests that if `time_viewer=False` then the bug will still be there. It seems like we Probably need an equivalent change in very similar code in `_pyvista.py`, or some other fix, or maybe this belongs at the `Brain` level?

Reply:
You're totally right. And this code in Brain can be removed since `self.show()` calls `self._renderer.show()`. I'll update on `_pyvista`

Reply:
... and cannot be removed since `ensure_minimum_size` has a different version for Brain.